src/main/java/org/jarvis4ops/helper/SlackHelper.java

```diff
@@ -129,6 +129,13 @@ public String composeWipBreachedMsg(HashMap<String, String> jiraMaxWipCountMap)
 		}
 		return jiraMaxWipBreachedDetailMsgSb.toString();
 	}
+	
+	public String composeFoundWorkMsg(HashMap<String, String> jiraFoundWorkCountMap) {
+		StringBuilder jiraFoundWorkDetailMsgSb = new StringBuilder();
+		int count = Integer.parseInt(jiraFoundWorkCountMap.get("FoundWorkCount"));
+		jiraFoundWorkDetailMsgSb.append(" " + count + "- new item" );
```

Comment:
Can we move this string to slack properties file aka pick through constant (saves heap consumption) @cshar7